types/exit/index.d.ts

```diff
@@ -1,8 +1,8 @@
-// Type definitions for exit 0.1.2
+// Type definitions for exit 0.1
 // Project: https://github.com/cowboy/node-exit
 // Definitions by: Bart van der Schoor <https://github.com/Bartvds>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-
 declare function exit(code: number): void;
+declare namespace exit { }
```

Comment:
No. Please read https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this